api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java

```diff
@@ -137,6 +145,11 @@ public Obs toOpenmrsType(Obs existingObs, Observation observation, Supplier<Obs>
 			interpretationTranslator.toOpenmrsType(existingObs, observation.getInterpretation().get(0));
 		}
 		existingObs.setDateChanged(observation.getMeta().getLastUpdated());
+		existingObs.setDateCreated(observation.getIssued());
```

Comment:
We should not being doing this. Let the `dateCreated` (and `dateChanged`) fields get updated by the system, not by our code.

Reply:
@ibacher I'm not sure I understand this, do you mean like we shouldn't be mapping openmrs fields?

Reply:
I just mean we shouldn't be updating `dateCreated` and `dateUpdated` based on values supplied via the FHIR interface. They should basically be read-only fields.

Reply:
got it